src/patternfly/components/ChipGroup/chip-group.scss

```diff
@@ -24,24 +25,28 @@
   display: inline-flex;
   flex-wrap: wrap;
   align-items: center;
-  padding: var(--pf-c-chip-group--PaddingTop) var(--pf-c-chip-group--PaddingRight) var(--pf-c-chip-group--PaddingBottom) var(--pf-c-chip-group--PaddingLeft);
-  border-radius: var(--pf-c-chip-group--BorderRadius);
+
 
   &.pf-m-toolbar {
-    background-color: var(--pf-c-chip-group--m-toolbar--BackgroundColor);
-  }
+    > li {
+      padding: var(--pf-c-chip-group--PaddingTop) var(--pf-c-chip-group--PaddingRight) var(--pf-c-chip-group--PaddingBottom) var(--pf-c-chip-group--PaddingLeft);
+      background-color: var(--pf-c-chip-group--m-toolbar--BackgroundColor);
+      border-radius: var(--pf-c-chip-group--BorderRadius);
 
-  > * {
-    margin-top: var(--pf-c-chip--MarginTop);
-    margin-bottom: var(--pf-c-chip--MarginBottom);
+      .pf-c-chip-group__label,
+      .pf-c-chip {
+        margin-top: var(--pf-c-chip--MarginTop);
+        margin-bottom: var(--pf-c-chip--MarginBottom);
+      }
 
-    &:not(:last-child),
-    &:not(:only-child) {
-      margin-right: var(--pf-c-chip--MarginRight);
+      &:not(:last-child) {
+        margin-right: var(--pf-c-chip-group__li--MarginRight);
+      }
```

Comment:
Can you look at all the vars in the `.pf-m-toolbar` block and make sure they're named properly? Most don't have `--m-toolbar` in them and we're using `--pf-c-chip--*` vars here, too.